vmngclient/api/templates/device_template/device_template.py

```diff
@@ -14,7 +17,7 @@ class GeneralTemplate(BaseModel):
     class Config:
         arbitrary_types_allowed = True
 
-    name: str
+    name: str = ""
     subTemplates: List[GeneralTemplate] = []
 
     templateId: str = ""
```

Comment:
Consider adding validator if one of  `name` and `templateId` is a mandatory field for `Config`.